go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/deepfence/FlowMeter
+module github.com/Prathu122/FlowMeter
```

Comment:
Can you change it back to deepfence?

Reply:
Ok I'll do it. I didn't notice there were problems due to the code. I'll change it